### DISCUSSION TITLE
Error when using vectorSemanticHybrid as azure search query type

### DISCUSSION THREAD

**Discussion by morganlangley98** [microsoft/sample-app-aoai-chatGPT#1115] [General]
I'm encountering an error when using Azure Cognitive Search in Azure OpenAI. 

The error message states: Azure Search Error: 400, message='Server responded with status 400. Error message: {"error":{"code":"InvalidRequestParameter","message":"Unknown semantic configuration 'default'."}}.

I have semantic search configured in my index and you can see the search.rerankerScore. The app works perfectly with vectorSimpleHybrid as the azure search query type.

I am using the api-version=2024-03-01-preview for the Azure OpenAI, no documentation indicating it would cause this problem though.

Has anyone else come across this problem? Any help much appreciated.

Cheers,
Morgan

**Comment by misslapina123** [comment]
What location is your AI Search in?

**Reply by morganlangley98** [reply]
UK South, have you had a similar problem? Seems there are a few bugs about at the moment.

**Reply by misslapina123** [reply]
You can try if the problem is still there if you move it to West or North Europe. We had a similar issue when having AI Search in Sweden Central, it was resolved by changing the location.
